src/Sylius/Bundle/RbacBundle/Form/Exception/UnexpectedTypeException.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\RbacBundle\Form\Exception;
+
+use Symfony\Component\Form\Exception\InvalidArgumentException;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class UnexpectedTypeException extends InvalidArgumentException
```

Comment:
This class is redundant. See `Sylius\Component\Resource\Exception\UnexpectedTypeException`